app/views/unattended/report_templates/host_-_last_checkin.erb

```diff
@@ -0,0 +1,28 @@
+<%#
+name: Host - Last Checkin
+snippet: false
+template_inputs:
+- name: Search
```

Comment:
The name is bit missleading I can't come up with a better one though xD
Like this it sounds like I can put whatever I want ( only `os = RedHat` ) and it will still give me only inactive hosts.